wiki/Getting-Started.md

```diff
@@ -178,8 +178,10 @@ To do that, put files into the correct directories according to this table.
 | `resources/niri-session` | `/usr/bin/` |
 | `resources/niri.desktop` | `/usr/share/wayland-sessions/` |
 | `resources/niri-portals.conf` | `/usr/share/xdg-desktop-portal/` |
-| `resources/niri.service` | `/usr/lib/systemd/user/` |
-| `resources/niri-shutdown.target` | `/usr/lib/systemd/user/` |
+| `resources/niri.service` (systemd) | `/usr/lib/systemd/user/` |
+| `resources/niri-shutdown.target` (systemd) | `/usr/lib/systemd/user/` |
+| `resources/dinit/niri` (dinit) | `/usr/lib/dinit.d/user/` |
+| `resources/dinit/niri-shutdown` (dinit) | `/usr/lib/dinit.d/user/` |
```

Comment:
At https://davmac.org/projects/dinit/man-pages-html/dinit.8.html is says: 

> Dinit reads service descriptions from files located in a service description directory, normally one of /etc/dinit.d, /run/dinit.d, /usr/local/lib/dinit.d and /lib/dinit.d for the system instance or $XDG_CONFIG_HOME/dinit.d, $HOME/.config/dinit.d, /etc/dinit.d/user, /usr/lib/dinit.d/user and /usr/local/lib/dinit.d/user when run as a user process. See SERVICE DESCRIPTION FILES for details of the service description format.

In packaging, it is better to use `/usr/lib/dinit.d/user`, and with manual system-wide installation `/etc/dinit.d/user`. 

There is also a way to install it for the current user only (`$XDG_CONFIG_HOME/dinit.d`). I didn't include it in the table because I was only writing the equivalent files to installation with systemd.